content/webapp/components/NoEvents/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+import { font } from '@weco/common/utils/classnames';
+import ButtonSolidLink from '@weco/common/views/components/Buttons/Buttons.SolidLink';
+import Space from '@weco/common/views/components/styled/Space';
+
+import { QuestionMark } from './images';
+
+type Props = {
+  period: 'past' | 'future';
+  hasFilters: boolean;
+};
+
+const NoResultsWrap = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+const ClearFiltersText = styled(Space).attrs({
+  as: 'p',
+  $v: { size: 'xl', properties: ['margin-bottom'] },
+})`
+  color: ${props => props.theme.color('neutral.600')};
+`;
+
+const NoEvents: FunctionComponent<Props> = ({ period, hasFilters }) => {
+  const isPast = period === 'past';
```

Comment:
super nitpick but could be `isPastListing` to align with other places it's used (`isPast` tends to be for a single event/exhibition, so just thinking of future keyword searches)

Reply:
the prop can probably just be `isPastListing` because I'm not doing anything else with the `period` – will update